packages/app/components/Insights/components/InsightsDisplayOptions.tsx

```diff
@@ -1,9 +1,18 @@
-import { ReactElement } from "react";
+import React, { ReactElement } from "react";
 import { API } from "@koh/api-client";
 import useSWR from "swr";
-import { Form, Switch } from "antd";
+import { Divider, Switch } from "antd";
 import { useProfile } from "../../../hooks/useProfile";
+import styled from "styled-components";
 
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-size: 18px;
+  align-items: center;
+  margin-bottom: 16px;
+`;
 interface InsightsDisplayOptionsProps {
   toggleInsightOn: (insightName: string) => void;
   toggleInsightOff: (insightName: string) => void;
```

Comment:
Good catch! Previously the toggleOff function was used to be able to toggle off an insight from the hide button on the insight itself but that made for a confusing UX so I removed it. That means they no longer need to be passed in as props. 

I just moved both the `toggleInsightOn` and `toggleInsightOff` functions into the InsightsDisplayOptions component instead of passing them in as props.